submissions/okkkko/js_oop/app.js

```diff
@@ -0,0 +1,87 @@
+const PLAYER_START_POSUTION = {x: 200, y: 380};
+const STEP_Y = 80;
+const STEP_X = 100;
+const FIRST_ROW = 60;
+const BUGS_START_POSUTION = {x:-200, y: [FIRST_ROW, FIRST_ROW + STEP_Y, FIRST_ROW + STEP_Y*2, FIRST_ROW + STEP_Y*3]};
```

Comment:
What you will do if you will need 100500 bugs, you will add y position one by one to this array ?
Maybe made some function which will calculate `y` position for all bugs automatically ?

Reply:
yes, I`ll fix it